test/Microsoft.Health.Fhir.R4.Tests.Integration/Microsoft.Health.Fhir.R4.Tests.Integration.csproj

```diff
@@ -6,7 +6,7 @@
     <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="3.1.8" />
     <PackageReference Include="Microsoft.Health.Test.Utilities" Version="1.0.0-master-20201008-2" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.7.0" />
-    <PackageReference Include="Microsoft.SqlServer.DACFx" Version="150.4573.2" />
+    <PackageReference Include="Microsoft.SqlServer.DACFx" Version="150.4897.1" />
```

Comment:
The version upgrade fixes the below error with the connectionString and eliminate the need to remove default value for MultipleActiveResultSets and TrustServerCertificate
  Error Message:
   System.ArgumentException : Keyword not supported: 'multiple active result sets'.
  Stack Trace:
     at System.Data.SqlClient.SqlConnectionStringBuilder.GetIndex(String keyword)
   at System.Data.SqlClient.SqlConnectionStringBuilder.set_Item(String keyword, Object value)
   at System.Data.Common.DbConnectionStringBuilder.set_ConnectionString(String value)
   at System.Data.SqlClient.SqlConnectionStringBuilder..ctor(String connectionString)
   at Microsoft.Data.Tools.Schema.Common.SqlClient.SqlConnectionFactory..ctor(String connectionString)
   at Microsoft.SqlServer.Dac.Compare.SchemaCompareDatabaseEndpoint..ctor(String connectionString)
   at Microsoft.Health.Fhir.Tests.Integration.Persistence.SqlServerFhirStorageTestHelper.CompareDatabaseSchemas(String databaseName1, String databaseName2) in D:\a\1\s\test\Microsoft.Health.Fhir.Shared.Tests.Integration\Persistence\SqlServerFhirStorageTestHelper.cs:line 107
   at Microsoft.Health.Fhir.Tests.Integration.Persistence.SqlServerSchemaUpgradeTests.GivenTwoSchemaInitializationMethods_WhenCreatingTwoDatabases_BothSchemasShouldBeEquivalent() in D:\a\1\s\test\Microsoft.Health.Fhir.Shared.Tests.Integration\Persistence\SqlServerSchemaUpgradeTests.cs:line 38
   at Microsoft.Health.Fhir.Tests.Integration.Persistence.SqlServerSchemaUpgradeTests.GivenTwoSchemaInitializationMethods_WhenCreatingTwoDatabases_BothSchemasShouldBeEquivalent() in D:\a\1\s\test\Microsoft.Health.Fhir.Shared.Tests.Integration\Persistence\SqlServerSchemaUpgradeTests.cs:line 44
--- End of stack trace from previous location where exception was thrown ---